mycroft/skills/intent_service.py

```diff
@@ -136,8 +137,9 @@ def __init__(self, bus):
 
     @property
     def registered_intents(self):
+        lang = _get_message_lang()
```

Comment:
Possibly a minor efficiency and stability gain by putting a `dig_for_message` call here, since its a couple frames fewer to look back through (less to iterate and lower chance the message is outside our maximum traceback range)

Reply:
that would be duplicating code for a negligible gain, i prefer to keep it more logical and readable, the dig call happens when needed in the method where it is relevant